beep/generate_protocol.py

```diff
@@ -48,9 +48,9 @@
 from monty.serialization import loadfn
 
 from beep import logger, __version__
-from beep.protocol import PROCEDURE_TEMPLATE_DIR
+from beep.protocol import PROCEDURE_TEMPLATE_DIR, BIOLOGIC_TEMPLATE_DIR
 from beep.protocol.maccor import Procedure
-
+from beep.protocol.biologic import Settings
```

Comment:
Just to confirm, Biologic:Settings :: Maccor:Procedure :: Arbin:Schedule ?

Reply:
Yup!